Sources/BugsnagPerformance/Private/OtlpTraceEncoding.mm

```diff
@@ -17,8 +17,8 @@
     return [NSString stringWithFormat:@"%016llx", spanId];
 }
 
-static NSString * EncodeTraceId(TraceId const &traceId) {
-    return [NSString stringWithFormat:@"%016llx%016llx", traceId.hi, traceId.lo];
+static NSString * EncodeTraceId(uint64_t const &traceIdHi, uint64_t const &traceIdLo) {
```

Comment:
Actually we don't need this to be a reference anymore since they are now primitives.